server/tournaments/tournament_game.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+from enum import Enum
+
+from server.players import Player
+
+
+class TournamentGameState(Enum):
+    SCHEDULED = 1
+    CONFIRMATION_PENDING = 2
+    STARTING = 3
+    RUNNING = 4
+    PLAYER_NOT_ONLINE = 5
+    PLAYER_NOT_IDLE = 6
+    PLAYER_NOT_CONFIRMING = 7
+    PLAYER_NOT_STARTING = 8
+    PLAYER_NOT_CONNECTING = 9
+
+    def is_done(self):
+        return self not in [
+            TournamentGameState.SCHEDULED,
+            TournamentGameState.CONFIRMATION_PENDING,
+            TournamentGameState.STARTING
+        ]
```

Comment:
I'd probably use a set.
```suggestion
        return self not in {
            TournamentGameState.SCHEDULED,
            TournamentGameState.CONFIRMATION_PENDING,
            TournamentGameState.STARTING
        }
```